drivers/SmartThings/zigbee-smoke-detector/capabilities/lifeTimeReport.yaml

```diff
@@ -0,0 +1,22 @@
+id: stse.lifeTimeReport
+version: 1
+status: proposed
+name: Life Time Report
+ephemeral: false
+attributes:
+  lifeTimeState:
+    schema:
+      type: object
+      properties:
+        value:
+          type: string
+          enum:
+            - normal
+            - beAboutToEnd
```

Comment:
this value is never used

is it intended to be used by future devices?

Reply:
> this value is never used
> 
> is it intended to be used by future devices?

Yes.